server-subscriptions-as3/README.md

```diff
@@ -0,0 +1,20 @@
+# Subscriptions in Apollo Server v3
+
+This example demonstrates a basic subscription operation in Apollo Server. [See the docs on subscriptions](https://www.apollographql.com/docs/apollo-server/data/subscriptions)
+
+The example server exposes one subscription (`numberIncremented`) that returns an integer that's incremented on the server every second.
+
+After you start up this server, you can test out running a subscription with the Apollo Studio Explorer or GraphQL Playground. You'll see the subscription's value update every second.
```

Comment:
perhaps suggest a specific subscription to run?

btw I ran into an issue where unnamed subscriptions don't work in explorer. https://github.com/mdg-private/studio-ui/pull/4357/files#r661810202

Reply:
(so you should suggest a subscription with a name)

Reply:
Yeah. I'm not going to call out that it's currently broken since presumably will fix it, but I will give them a subscription to copy/paste.